content/docs/1.10.0/advanced-resources/rebuilding/offline-replica-rebuilding.md

```diff
@@ -29,6 +29,19 @@ Starting with v1.9.0, Longhorn supports offline replica rebuilding, allowing deg
 - When triggered, Longhorn attaches the volume without activating the frontend, rebuilds any missing replicas, and then detaches the volume upon completion.
 - This process can be interrupted if the workload scales up.
 
+## Resource Constraints
+
+Offline replica rebuilding will be cancelled or prevented from starting if there are insufficient resources available in the cluster.
+
+- All instance managers on schedulable nodes must be ready.
+- The scheduler verifies that there is an available disk with sufficient space on a schedulable node for rebuilding a replica.
```

Comment:
Can you reorganize the sentences? The two items don't have strong relationship with 
```
Offline replica rebuilding will be cancelled or prevented from starting if there are insufficient resources available in the cluster.
```